dnsmasq/apparmor.txt

```diff
@@ -25,6 +25,11 @@ profile dnsmasq flags=(attach_disconnected,mediate_deleted) {
   /dev/tty rw,
   /tmp/* rw,
 
-  /run.sh rix,
-  /data/** r,
+  /etc/s6/** ix,
+  /run/s6/** ix,
+  /etc/services.d/** rwix,
+  /etc/cont-init.d/** rwix,
+  /etc/cont-finish.d/** rwix,
+
+  /run/** rwk,
```

Comment:
Make sure they cover this basic: https://developers.home-assistant.io/docs/add-ons/presentation#apparmor